communication/src/main/java/org/jboss/da/communication/aprox/model/VersionResponse.java

```diff
@@ -0,0 +1,59 @@
+package org.jboss.da.communication.aprox.model;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "metadata")
+public class VersionResponse {
+
```

Comment:
John told us that galley supports work with maven metadata but when we consider that we need to download it and parse it I think that this is the simplest way to do it.